src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -459,7 +465,7 @@ void VtolAttitudeControl::fill_fw_att_control_output()
 	_actuators_out_0.control[3] = _actuators_fw_in.control[3];
 	/*controls for the elevons */
 	_actuators_out_1.control[0] = -_actuators_fw_in.control[0];	// roll elevon
-	_actuators_out_1.control[1] = _actuators_fw_in.control[1];	// pitch elevon
+	_actuators_out_1.control[1] = _actuators_fw_in.control[1] + _params.fw_pitch_trim/100.0f;	// pitch elevon
```

Comment:
I missed the division by 100 here - is this intended to be percent? Given that our whole architecture runs from -1 to +1, just using the native unit will probably be less confusing.